customize_image.sh

```diff
@@ -138,6 +141,7 @@ losetup "$LOOPDEVICE" "$1"
 partprobe "$LOOPDEVICE"
 mount "$LOOPDEVICE"p2 "$IMAGEDIR"
 mount "$LOOPDEVICE"p1 "$IMAGEDIR/boot"
+mount --bind /etc/resolv.conf /mnt/etc/resolv.conf
```

Comment:
```suggestion
mount --bind /etc/resolv.conf "$IMAGEDIR/etc/resolv.conf"
```

Reply:
Good catch, thanks. I've updated the PR and did a force-push